### PR TITLE
feat: [#4446] - E2E tests for managing genes and tissues

### PR DESCRIPTION
**Pull request by charles-testco**
## Reason for Change

- #4446

- add
New tests for adding tissues
New tests for adding and removing genes

## Notes for Reviewer
Firefox excluded because clipboard-read permission is support
Should be reviewed after previous PRs merged

### CODE REVIEW

**Comment by tihuan** [comment]
IIRC our codecov only tracks python files, so I don't think it's relevant to us!

**Comment by charles-testco** [comment]
@tihuan  @atarashansky thanks for review

**Review by tihuan — APPROVED** [review]
Great PR, @charles-testco ! I also verified that tets passed locally 🎉  (since WMG tests don't run in PRs) 

Just a few tiny comments and an idea to improve README

PTAL thank you!

File: frontend/playwright.config.ts [hunk 1]
```diff
@@ -66,6 +73,7 @@ const config: PlaywrightTestConfig = {
       use: {
         ...devices["Desktop Chrome"],
         userAgent: devices["Desktop Chrome"].userAgent + CZI_CHECKER,
+        permissions: [CLIPBOARD_READ, CLIPBOARD_WRITE],
```
[the remark this answers is not in the document]
Comment (charles-testco) [reply frontend/playwright.config.ts:81]: Added 
CLIPBOARD_PERMISSIONS = ["clipboard-read", "clipboard-write"]

File: frontend/tests/features/wheresMyGene/gene.test.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+describe("Manage gene tests", () => {
+  test("Should select gene using keyboard arrow key to select", async ({
+    page,
+  }) => {
+    const GENE = "TNMD";
+    await goToWMG(page);
+    // click +Tissue button
+    await page.getByTestId(ADD_GENE_BTN).click();
+    await page.keyboard.press("ArrowDown");
```
[the remark this answers is not in the document]
Comment (charles-testco) [reply frontend/tests/features/wheresMyGene/gene.test.ts:19]: Oh, I didn't know that. Great

**Pull request merged**